cypress-tests/cypress/support/commands.js

```diff
@@ -671,6 +682,14 @@ Cypress.Commands.add("handleRedirection", (globalState, expected_redirection) =>
             cy.get('#test-source-authorize-3ds').click();
           })
       })
+  } else if (connectorId == "adyen") {
```

Comment:
Get the connectorId from the globalstate
`if (globalState.get("connectorId") == "stripe") `
Because while running all the collection it will break

Reply:
This has nothing to do with the collection breaking. Verified with @likhinbopanna as well. They're running good.

Also, `handleRedirection` is called from the beginning. It is not that a specific part of it is run. So, it will always have the updated `connectorId` no matter what.